src/style.css

```diff
@@ -1,3 +1,68 @@
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
```

Comment:
Ensure that the 'Roboto' font is correctly imported in the HTML file to avoid fallback to the default serif font.